test/test_model.py

```diff
@@ -3,9 +3,11 @@
 import os
 import shutil
 import unittest
+from unittest.mock import Mock
```

Comment:
does unittest.mock work with PyTest?

Reply:
Yes. Here is also one example from ArviZ https://github.com/arviz-devs/arviz/blob/master/arviz/tests/test_utils.py